packages/camera/camera_avfoundation/ios/Classes/CameraProperties.h

```diff
@@ -30,49 +32,24 @@ extern AVCaptureFlashMode FLTGetAVCaptureFlashModeForFLTFlashMode(FLTFlashMode m
 
 #pragma mark - exposure mode
 
-/// Represents camera's exposure mode. Mirrors ExposureMode in camera.dart.
-typedef NS_ENUM(NSInteger, FLTExposureMode) {
-  FLTExposureModeAuto,
-  FLTExposureModeLocked,
-  // This should never occur; it indicates an unknown value was received over
-  // the platform channel.
-  FLTExposureModeInvalid,
-};
-
-/// Gets a string representation of exposure mode.
-/// @param mode exposure mode
-extern NSString *FLTGetStringForFLTExposureMode(FLTExposureMode mode);
-
-/// Gets FLTExposureMode from its string representation.
-/// @param mode a string representation of the FLTExposureMode.
-extern FLTExposureMode FLTGetFLTExposureModeForString(NSString *mode);
+/// Gets FCPPlatformExposureMode from its string representation.
+/// @param mode a string representation of the exposure mode.
+extern FCPPlatformExposureMode FCPGetExposureModeForString(NSString *mode);
```

Comment:
These are temporary methods; there are Dart->host calls that send exposure and focus modes, so until I convert those in a follow-up we need to be able to convert method channel strings to the Pigeon enum that replaced the ones removed here.